octree/include/pcl/octree/octree_container.h

```diff
@@ -212,16 +192,15 @@ namespace pcl
         }
 
         /** \brief Empty constructor. */
-        OctreeContainerPointIndex (const OctreeContainerPointIndex& source) :
-            data_ (source.data_)
+        OctreeContainerPointIndex (const OctreeContainerPointIndex& source)
+        : OctreeContainerBase (source)
+        , data_ (source.data_)
```

Comment:
This is an explicit invocation of the default copy ctor. It can be erased I believe.